contrib/ansible/vagrant/openstack_config.yml

```diff
@@ -0,0 +1,12 @@
+os_username: eparis
```

Comment:
Do you want to commit this info? Should we add to gitignore once it is an example file?